docs/content/preview/yugabyte-cloud/release-notes.md

```diff
@@ -1,19 +1,24 @@
 ---
-title: What's new in YugabyteDB Managed
-linkTitle: What's new
-description: YugabyteDB Managed release notes and known issues.
-headcontent: New features and known issues
+title: Change log for YugabyteDB Managed
+headerTitle: Change log
```

Comment:
I prefer the old title for the header- What's new in YugabyteDB Managed because of multiple occurrences of change log on the page.